src/index.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="{{.Locale}}">
```

Comment:
Nevertheless, within the mobile application version we use the `navigator` object, see https://github.com/cozy/cozy-files-v3/blob/master/mobile/src/main.jsx#L25